add new integration: ovhcloud_ai_endpoints (#45472)
home-assistant/home-assistant.github.io: source/_integrations/ovhcloud_ai_endpoints.markdown

ha_platforms:
  - conversation
ha_quality_scale: bronze
related:
  - docs: /voice_control/voice_remote_expose_devices/
    title: Exposing entities to Assist
  - docs: /voice_control/assist_create_open_ai_personality/
    title: Create an AI personality
---

The **OVHcloud AI Endpoints** {% term integration %} adds a conversation agent powered by [OVHcloud AI Endpoints](https://endpoints.ai.cloud.ovh.net/) to Home Assistant. OVHcloud AI Endpoints is a serverless inference platform that gives you access to a curated catalog of open-weight large language models hosted in Europe.

Controlling Home Assistant is done by providing the AI access to the Assist API of Home Assistant. You can control what devices and entities it can access from the {% my voice_assistants title="exposed entities page" %}. The AI can provide you with information about your devices and control them.

This integration is useful if you want to keep your conversation data within a European cloud provider, or if you want to use a specific open-weight model from the OVHcloud AI Endpoints catalog.

## Prerequisites

- You need an [OVHcloud account](https://www.ovhcloud.com/) with access to the AI Endpoints product.

> - This is a paid service. We advise you to monitor your usage and costs in the [OVHcloud Control Panel](https://manager.eu.ovhcloud.com/) to avoid unwanted charges.

Reviewer: ```suggestion
- This is a paid service. Monitor your usage and costs in the [OVHcloud Control Panel](https://manager.eu.ovhcloud.com/) to avoid unwanted charges.
```

### Generating an API key

> The API key is used to authenticate requests to OVHcloud AI Endpoints. To generate an API key, follow the [official guide](https://help.ovhcloud.com/csm/en-public-cloud-ai-endpoints-getting-started?id=kb_article_view&sysparm_article=KB0065403).

Author: ```suggestion
The API key is used to authenticate requests to OVHcloud AI Endpoints. To generate an API key, follow the [official guide](https://docs.ovhcloud.com/en/guides/public-cloud/ai-machine-learning/ai-endpoints-getting-started).
```

> The rough steps are outlined below:

Reviewer: ```suggestion
The guide includes the following steps:
```

1. Log in to the [OVHcloud Control Panel](https://manager.eu.ovhcloud.com/#/hub/).
2. Go to the **Public Cloud** section and select your project or create a new one.
3. In the left menu, under **AI & Machine Learning**, select **AI Endpoints**.
4. Open the **API keys** tab and create a new API key.
5. Copy the generated key. You'll use it to configure the integration.

{% include integrations/config_flow.md %}

{% configuration_basic %}
API key:
  description: "API key from OVHcloud AI Endpoints used to authenticate requests."
{% endconfiguration_basic %}

{% include integrations/option_flow.md %}

## Configuration options

> 

Reviewer: ```suggestion
```
The title is already part of the include.

Author: Didn’t even see that when proofreading 🤦🏻‍♂️

Reviewer: I know the feeling.
